src/compiler/sys/fetch/fetch-utils.ts

```diff
@@ -2,8 +2,16 @@ import type * as d from '../../../declarations';
 import { isCommonDirModuleFile, isTsFile, isTsxFile } from '../resolve/resolve-utils';
 import { isFunction, normalizePath } from '@utils';
 
+/**
+ * A fetch wrapper which dispatches to `sys.fetch` if present, and otherwise
+ * uses `global.fetch`.
+ *
+ * @param sys a compiler system object
+ * @param input a `RequestInfo` object
+ * @param init an optional `RequestInit` object
+ * @returns a Promise wrapping a response
+ */
 export const httpFetch = (sys: d.CompilerSystem, input: RequestInfo, init?: RequestInit): Promise<Response> => {
-  console.trace(input);
```

Comment:
there's no reason for us to leave this lurking in here, right?

Reply:
I hope not anyway!

Reply:
I hope not 😆